avm/res/network/network-watcher/tests/e2e/defaults/main.test.bicep

```diff
@@ -0,0 +1,51 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults'
+metadata description = 'This instance deploys the module with the minimum set of required parameters.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'NetworkWatcherRG' // Note, this is the default NetworkWatcher resource group. Do not change.
+
+@description('Optional. The location to deploy resources to.')
+#disable-next-line no-unused-params // A rotation location cannot be used for this test as NetworkWatcher Resource Groups are created for any test location automatically
+param resourceLocation string = deployment().location
+
+@description('Optional. The static location of the resource group & resources.') // Note, we set the location of the NetworkWatcherRG to avoid conflicts with the already existing NetworkWatcherRG
+param tempLocation string = 'uksouth'
```

Comment:
Hey @ChrisSidebotham, as pointed out several times internally, could you please advice on the locations to set for the tests? 
Given the deployed compute for some of the tests + the point that we should not overlap with our other test resources means we need alternatives ;)